src/ontology/cl-edit.owl

```diff
@@ -3094,6 +3095,7 @@ Declaration(Class(obo:UBERON_0002339))
 Declaration(Class(obo:UBERON_0002548))
 Declaration(Class(obo:UBERON_0002707))
 Declaration(Class(obo:UBERON_0003209))
+Declaration(Class(obo:UBERON_0003677))
```

Comment:
Need to import this term, right?

Reply:
I will refresh imports for the CL release.

Reply:
you could do it here, and not do later.